ext/opentelemetry-ext-jinja2/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from opentelemetry.ext.jinja2 import Jinja2Instrumentor
+
+_INSTRUMENTOR = Jinja2Instrumentor()
```

Comment:
BaseInstrumentor ensures any implementor is a singleton, I believe this global variable is not necessary here.